allennlp/semparse/contexts/table_question_knowledge_graph.py

```diff
@@ -78,6 +81,21 @@ class TableQuestionKnowledgeGraph(KnowledgeGraph):
     Additionally, when we encounter cells that can be split, we create ``fb:part.[something]``
     entities, also without any neighbors.
     """
+    def __init__(self,
+                 entities: Set[str],
+                 neighbors: Dict[str, List[str]],
+                 entity_text: Dict[str, str],
+                 question_tokens: List[Token]) -> None:
+        super(TableQuestionKnowledgeGraph, self).__init__(entities, neighbors, entity_text)
+        self.question_tokens = question_tokens
+        self._entity_prefixes: Dict[str, List[str]] = defaultdict(list)
+        for entity, text in self.entity_text.items():
+            parts = text.split()
+            if len(parts) < 1:
```

Comment:
`if not parts:`.  Or, alternatively, `if not text:`, above.

Reply:
Done.